src/main/types/workspaceIssuesDetails.ts

```diff
@@ -30,6 +30,26 @@ export class ScanResults {
         return workspaceIssuesDetails;
     }
 
+    /**
+     * Get the oldest timestamp from all its results
+     */
+    public get oldestScanTimestamp(): number | undefined {
```

Comment:
Looks like we already have oldestScanTimestamp in IssuesRootTreeNode.
Can we reuse the code?